src/main/java/Ui.java

```diff
@@ -0,0 +1,73 @@
+import java.util.ArrayList;
+
+/**
+ * Encapsulates the User Interface operations. It is responsible of all interaction with the user.
+ */
+
+public class Ui {
+
+    public void welcome() {
```

Comment:
Since all the methods in this class are stateless, they could all be made `static` instead. At present there is no functionality gained from instantiating a `Ui` object.